src/server/scripts/EasternKingdoms/zone_undercity.cpp

```diff
@@ -2053,7 +2053,8 @@ class npc_varian_wrynn : public CreatureScript
                     case EVENT_AGGRO_JAINA:
                         if (me->GetVictim())
                             if (Creature* jaina = ObjectAccessor::GetCreature(*me, jainaGUID))
-                                jaina->AI()->AttackStart(me->GetVictim()); DoCast(me, SPELL_THUNDER);
+                                jaina->AI()->AttackStart(me->GetVictim());
+                        DoCast(me, SPELL_THUNDER);
```

Comment:
Note: this might be a bug. Here I'm just changing the indentation to make it clear that not all statements are part of the IF block. **I'm NOT changing the behaviour**.

Reply:
I recommend adding brackets to do not make this misunderstanding again.

Reply:
done